service/iot-service-samples/device-twin-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/DeviceTwinSample.java

```diff
@@ -49,22 +49,22 @@ public static void main(String[] args) throws Exception
         {
             // Manage complete twin
             // ============================== get initial twin properties =============================
-            Twin device = getInitialState(twinClient, deviceId);
```

Comment:
We don't need to cache the device twin and then pass it in each twin operation method below. When we did so, we were using the **existing** `Twin` instance, as a result, a replace call with the cached device twin would also update the unintended properties. We should've been created a new `Twin` instance each time when we do a twin operation.